src/openzaak/components/zaken/migrations/0009_remove_zaakobject__etag.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
+# Generated by Django 3.2.13 on 2022-04-22 13:34
+
+from django.db import migrations
```

Comment:
Can't this migration be squashed with `0008`?

Reply:
sure